app/utils/UnloadingFindingsAnswersHelper.scala

```diff
@@ -38,7 +39,7 @@ class UnloadingFindingsAnswersHelper(userAnswers: UserAnswers, referenceDataServ
 
   def buildVehicleNationalityRow(index: Index): Future[Option[SummaryListRow]] =
     (for {
-      x <- OptionT.fromOption[Future](userAnswers.getIE043(VehicleRegistrationCountryPage(index)))
+      x <- OptionT.fromOption[Future](userAnswers.getIE043(CountryPage(index)))
```

Comment:
could we change this to be `userAnswers.get(CountryPage(index))` please - changing the getIE043 to get